roles/etcd/tasks/gen_certs_script.yml

```diff
@@ -83,7 +83,6 @@
         '{{ etcd_cert_dir }}/member-{{ node }}-key.pem',
         {% endfor %}]"
   delegate_to: "{{ groups['etcd'][0] }}"
-  run_once: yes
```

Comment:
at this place run_once is worked fine.

Put it back in place

etcd_master_certs used only in next task.
and both of these tasks executed when `    - inventory_hostname in groups['etcd']	`

Reply:
sorry. I'm confused